cfpb/sbl-frontend
# [e2e] unavailableApis.spec - Refactor + add extended timeouts

Part of #987 

## Changes

- [e2e] Unavailable APIs
  - Refactored to make the tests easier to read and reduce code duplication (`verifyApiBlockThenUnblock()`)
  - Added timeouts to enable test to consistently pass

## How to test this PR

1. Start `sbl-frontend`: `yarn start`
2. Open Playwright: `yarn playwright test --ui --workers 1`
3. Run test `filing-app` -> `unavailableApis.spec.ts` twice
4. Confirm it passes consistently

## Screenshots

<img width="371" alt="Screenshot 2024-10-16 at 3 19 23 PM" src="https://github.com/user-attachments/assets/844013ed-8bd2-43e4-bbc4-5e65e8e23127">

<img width="835" alt="Screenshot 2024-10-16 at 3 22 14 PM" src="https://github.com/user-attachments/assets/56e9f645-feb1-478c-8601-9a83f7ad533c">

## Notes
- TODO (Future PR): Utilize `e2e/utils/timeoutConstants.ts` in more tests to reduce one-off timeout values
